hyperstone/plugins/loaders/pe.py

```diff
@@ -130,18 +138,53 @@ class PELoader(Plugin):
     Loads PE files from disk.
     interact() can be used to map a PE file.
     """
+
     _INTERACT_TYPE = PELoaderInfo
-    _IAT_HOOK_SEP = '@'
+    _IAT_HOOK_SEP = "@"
+
+    def __init__(self, *files: PELoaderInfo, dll_search_paths: list[str] = None):
```

Comment:
`List[str]`, not `list[str]`, due to `List` being `typing.List` and implementing `__getitem__`, on some interpreters `list[str]` might error